test/functional/TestUtilities/src/org/openj9/test/util/PlatformInfo.java

```diff
@@ -33,6 +33,10 @@ public static boolean isMacOS() {
 		return ((null != osName) && osName.startsWith("Mac OS"));
 	}
 
+	public static boolean isOpenJ9() {
+		return System.getProperty("java.vm.vendor").contains("OpenJ9");
```

Comment:
Not sure how this is intended to be used, but note that IBM uses OpenJ9 but sets `java.vm.vendor = IBM Corporation`

Reply:
Then purpose is to check for pure OpenJ9 builds, as opposed to IBM builds which do not contain things like `jps`, `jstack`, etc.

Reply:
Just echo to Pete's comment, please provide a description about the usage scenario.
From code perspective, `System.getProperty("java.vm.vendor")` can be in class initializer to avoid repeating invocation through method `isOpenJ9()`.
By the way, this is a code change, is it really worth to skip `ci` though I can't eyeball any potential compilation issue?

Reply:
@JasonFengJ9 the ci testing just builds the VM, while this change is to test material

Reply:
- Could we get some clarification how this methold will be used?
- We are using `java.vm.name` for AUTO_DETECT
https://github.com/eclipse/openj9/blob/6bf7e0de73f5e1ce5daac1d709d07fca97733900/test/TestConfig/src/org/openj9/envInfo/JavaInfo.java#L102

Reply:
- Elaborated the comment
- Set the value at class initialization